package.json

```diff
@@ -36,9 +36,11 @@
     "url": "git://github.com/alexdavid/exprestive.git"
   },
   "scripts": {
+    "compress": "gulp",
     "feature-tests": "./node_modules/.bin/cucumber-js test/feature",
     "prefeature-tests": "gulp",
-    "prepublish": "gulp",
+    "postpublish": "git push && git push --tags",
```

Comment:
I guess it does allow `npm version` easier to undo since its not automatically pushed up. Okay, I'm fine with this